src/YoutubeIframe.js

```diff
@@ -191,6 +191,7 @@ const YoutubeIframe = (props, ref) => {
           return request.mainDocumentURL === 'about:blank';
         }}
         {...webViewProps}
+        bounces={webViewProps?.bounces ?? false}
```

Comment:
it would make more sense to specify this as 

```jsx
bounces={false}
{...webViewProps}
```

this will handle nullish values better (false by default, overridden by `webViewProps`) 

and is more readable :)

Reply:
You’re right, that is better than mine 👍

Reply:
I’ll commit in few hours :)